x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx

```diff
@@ -311,81 +301,56 @@ const TGridIntegratedComponent: React.FC<TGridIntegratedProps> = ({
                 )}
               </UpdatedFlexGroup>
 
-              <FullWidthFlexGroup
-                $visible={!graphEventId && graphOverlay == null}
```

Comment:
How to test `graphOverlay` and `graphEventId`? It was always undefined when I implemented this feature. But I see that it was recently introduced on master.

Reply:
@stephmilovic is that correct?